incubator/mariadb-cluster/templates/slave-statefulset.xml

```diff
@@ -127,25 +140,18 @@ spec:
         resources:
 {{ toYaml .Values.metrics.resources | indent 10 }}
 {{- end }}
-      volumes:
-      - name: data
-        emptyDir: {}
```

Comment:
I suggest using a `enptyDir` volume when persistence is not enabled making it possible to perform some initialization on the data in init-containers if the need ever arises.